tests/integration/modifiers/did-resize-test.js

```diff
@@ -1,40 +1,33 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { find, render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import sinon from 'sinon';
 
-let resizeCallback;
-let observeStub;
-let unobserveStub;
-let disconnectStub;
-let MockResizeObserver;
-
 module('Integration | Modifier | did-resize', function (hooks) {
   setupRenderingTest(hooks);
 
-  let resizeObserver;
+  let resizeCallback = null;
+  const observeStub = sinon.stub();
```

Comment:
I'd prefer to leave these as `let` for lines 11 through 15, since they are not conceptually being used as constants. I prescribe to the "prefer let" mode of thinking: https://madhatted.com/2016/1/25/let-it-be